packages/php/src/Metrics.php

```diff
@@ -239,25 +239,17 @@ public function constructPayload(string $log_id, Request $request, $response): a
     }
 
     /**
-     * @psalm-suppress TaintedInput
```

Comment:
We were originally suppressing taint warnings here but IIRC this was coming from another section that no longer is a problem after a Psalm upgrade a while back.

Aside from the `@psalm-taint-escape`  annotation to tell Psalm that we're handling this tainted input scenario fine there are no other tainted input exclusions in this codebase.